lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -426,6 +438,36 @@ def test_submission_show_answer(self):
         )
         self.assertEqual(json.loads(resp.content).get("success"), err_msg)
 
+    @override_settings(KEY_PREFIX=str(uuid.uuid4()))  # hack: we do this to ensure a unique cache per test
```

Comment:
So after reading a bit more, I don't think this actually works. Will look over the tests again...